mne_bids/meg_bids.py

```diff
@@ -362,6 +377,16 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
             else:
                 raise ValueError('"%s" already exists. Please set overwrite to'
                                  ' True.' % raw_file_bids)
+        else:
+            # ensure the sub-folder exists
+            if not os.path.exists(os.path.dirname(raw_file_bids)):
+                os.makedirs(os.path.dirname(raw_file_bids))
```

Comment:
maybe reuse this: https://github.com/mne-tools/mne-bids/blob/master/mne_bids/utils.py#L11-L29 ?

Reply:
oh yeah, I meant to do that a while ago but forgot, thanks!